lib/assignment_manager.rb

```diff
@@ -59,6 +59,16 @@ def claim_assignment(claimed_assignment)
     end
   end
 
+  def self.check_wiki_edu_discouraged_article(title:)
+    category = Category.find_by('article_titles LIKE ? AND name = ?', "%#{title}%",
+                                'Wiki_Edu_Discouraged_Articles')
+
+    if category&.source == ENV['blocked_assignment_category']
```

Comment:
`Category#source` is for indicating which type of category record it is (ie, where it pulls the article title data from). In this case, the `Category` record should have `category` as the source (because it should use a Wikipedia category), and the `name` is the name of the Wikipedia category (which should come from configuration in application.yml).

Reply:
Okay